src/main/java/com/yahoo/sketches/frequencies/ItemsSketch.java

```diff
@@ -550,9 +600,7 @@ public int compare(final Row<T> r1, final Row<T> r2) {
       }
     });
     
-    @SuppressWarnings("unchecked")
```

Comment:
Eliminating this unchecked warning is not a good idea.  There is no way to get rid of the warning, so it just creates noise. It can either be here or at the method level, but it needs to stay.